tests/foreman/virtwho/cli/test_esx_sca.py

```diff
@@ -502,3 +505,93 @@ def test_positive_remove_env_option(
         env_warning = f"Ignoring unknown configuration option \"{option}\""
         result = target_sat.execute(f'grep "{env_warning}" /var/log/messages')
         assert result.status == 1
+
+    @pytest.mark.tier2
+    def test_positive_rhsm_username_option(
+        self, module_sca_manifest_org, form_data_cli, target_sat
+    ):
+        """Verify rhsm_username options in the configure file"
+
+        :id: 2d0d3126-859f-4092-9196-a553bc1d3bd9
+
+        :expectedresults:
+            1. virt-who config belong to a same org with the same rhsm_username
+            2. virt-who config belong to a different org with the different rhsm_username
+            3. hypervisors and guest mapping info send by group of org and service account exist in rhsm.log
+            4. not delete all virt-who config service account virt_who_reporter_X exist
+            5. delete all virt-who config service account virt_who_reporter_X will be deleted
+
+
+
+        :customerscenario: true
+
+        :CaseImportance: Medium
+
+        :BZ: 2063218
+        """
+        # get the service account for the first virt-who config on module_sca_manifest_org
+        name_1 = form_data_cli['name']
+        virtwho_config_cli_1 = target_sat.cli.VirtWhoConfig.create(form_data_cli)[
+            'general-information'
+        ]
+        config_file_1 = get_configure_file(virtwho_config_cli_1['id'])
+        command = get_configure_command(virtwho_config_cli_1['id'], module_sca_manifest_org.name)
+        deploy_configure_by_command(
+            command, form_data_cli['hypervisor-type'], org=module_sca_manifest_org.label
+        )
+        rhsm_username_1 = get_configure_option('rhsm_username', config_file_1)
+
+        # create the second virt-who config and get the service account for the first virt-who config on module_sca_manifest_org
+        name_2 = gen_string('alpha')
+        form_data_cli['name'] = name_2
+        virtwho_config_cli_2 = target_sat.cli.VirtWhoConfig.create(form_data_cli)[
+            'general-information'
+        ]
+        config_file_2 = get_configure_file(virtwho_config_cli_2['id'])
+        command_2 = get_configure_command(virtwho_config_cli_2['id'], module_sca_manifest_org.name)
+        deploy_configure_by_command(
+            command_2, form_data_cli['hypervisor-type'], org=module_sca_manifest_org.label
+        )
+        rhsm_username_2 = get_configure_option('rhsm_username', config_file_2)
+        # verify the two service account belong to the same org
+        assert rhsm_username_1 == rhsm_username_2
+        # Create a different org and then create virt-who config
```

Comment:
This can also be part of the loop, I don't see any reason not to do all the asserts after.

Reply:
> This can also be part of the loop, I don't see any reason not to do all the asserts after.

@lhellebr Have already updated